### PR TITLE
Add rake task to repopulate Monthly Agggregations

### PR DESCRIPTION
**Pull request by chao-xian**
For the Content Data App (Data Warehouse).

### CODE REVIEW

**Comment by chao-xian** [comment]
[(2) Raise 2ndline alarms when the Monthly aggregations are not collected from Google](https://trello.com/c/7Z5SLv8A/1141-2-raise-2ndline-alarms-when-the-monthly-aggregations-are-not-collected-from-google)

File: source/manual/alerts/content-performance-manager-app-healthcheck-not-ok.html.md [hunk 1]
```diff
@@ -12,11 +12,20 @@ Please notify Data Informed Content via Slack `#govuk-data-informed` about the a
 
 If there is a health check error showing for Content Performance Manager, you can click on the alert to find out more details about what’s wrong. Here are the possible problems you may see:
 
+## ETL :: no aggregations of metrics for yesterday
+
+This means that [the ETL master process][1] that runs daily that creates aggregations of the metrics failed.
```
Comment (pmanrubia) [inline source/manual/alerts/content-performance-manager-app-healthcheck-not-ok.html.md]: Could you please be specific about the monthly aggregations (also in the title of the error message)?

Comment (chao-xian) [reply source/manual/alerts/content-performance-manager-app-healthcheck-not-ok.html.md]: Thanks @pmanrubia - [fixed](https://github.com/alphagov/govuk-developer-docs/commit/815cf09d5697482bae97e3523680e6904c35a62f)

**Comment by pmanrubia** [comment]
👍

**Pull request merged**